main/tests/server/src/com/google/refine/importers/XmlImportUtilitiesTests.java

```diff
@@ -623,41 +551,27 @@ public void processRecordsFromComplexXmlStructure() throws IOException {
     }
 
     // ----------------helpers-------------
-    public void loadSampleXml() {
+    public void loadSampleXml() throws Exception {
         loadData(XmlImporterTests.getSample());
     }
 
-    public void loadSampleJson() {
+    public void loadSampleJson() throws Exception {
```

Comment:
This method appears to be unused, but it doesn't throw any exceptions.

```suggestion
    public void loadSampleJson() {
```